composer.json

```diff
@@ -11,15 +11,15 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "naucon/form": "^3.1",
-        "symfony/config": "^3.2 || ^4.0",
-        "symfony/dependency-injection": "^3.2 || ^4.0",
-        "symfony/finder": "^3.2 || ^4.0",
-        "symfony/http-kernel": "^3.2 || ^4.0"
+        "php": ">=7.3",
+        "naucon/form": "*",
```

Comment:
We should avoid using a wildcard here. Wait for the new release Tag from naucon/form probably 4.0.0 or 3.2.0.
Suggestion 3.2.0 for this change and 4.0.0 for Symfony 6 and PHP 8.1 updates @naucon

Reply:
changed it to 3.2.0